src/main/kotlin/com/linchpino/core/service/TimeSlotService.kt

```diff
@@ -0,0 +1,23 @@
+package com.linchpino.core.service
+
+import com.linchpino.core.dto.AddTimeSlotsRequest
+import com.linchpino.core.dto.toAvailableMentorTimeSlot
+import com.linchpino.core.repository.AccountRepository
+import com.linchpino.core.repository.MentorTimeSlotRepository
+import com.linchpino.core.repository.findReferenceById
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+
+@Service
+@Transactional
+class TimeSlotService(private val accountRepository: AccountRepository, private val repository: MentorTimeSlotRepository) {
+
+
+    fun addTimeSlots(request: AddTimeSlotsRequest) {
+        val account = accountRepository.findReferenceById(request.mentorId)
```

Comment:
I created an account with the type job seeker after that I added some slot time for JOB seeker. The account type can be validated here!
![image](https://github.com/linchpino/linchpino-core/assets/6872871/abbe93fb-0e84-4575-a713-75785b0e0edc)

![image](https://github.com/linchpino/linchpino-core/assets/6872871/0d2f8106-ad18-46a8-bf58-910759e99d18)

Please write a test case for this scenario also. Thank you